src/main/java/org/isf/ward/gui/WardBrowser.java

```diff
@@ -320,31 +323,34 @@ public int getColumnCount() {
 		@Override
 		public Object getValueAt(int r, int c) {
 			Ward ward = pWard.get(r);
+			int i = 0;
 			if (c == 0) {
 				return ward.getCode();
 			} else if (c == -1) {
 				return ward;
-			} else if (c == 1) {
+			} else if (c == ++i) {
```

Comment:
same comment as above

Reply:
same explanation as above, it makes easier to modify in future